.github/workflows/pytest.yml

```diff
@@ -25,8 +25,8 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install pytest
+        python -m pip install pytest
     - name: Test dependencies
       run: |
-        pip install -e .[all]
-        pytest tests
+        python -m pip install -e .[test]
+        python -m pytest tests
```

Comment:
If your code is not in `src/` then this won't pick up the files as you installed them, but instead from the current directly. But using Flit means you don't typically have to worry about stuff being left behind when packaging something up.

Reply:
moved code to src